### ISSUE TITLE
Are the EXPOSURE keyword in the ARF and the observation the same?

### ISSUE THREAD

**Issue by dhuppenkothen** [dhuppenkothen/clarsach#14]
In real data, does the "EXPOSURE" keyword in the ARF and the observation contain the same number? They don't for our simulated data, but I'm wondering if that's every true in real data or whether that's just a special case related to our simulations?

Background: I've been thinking of moving the multiplication of `frac_exp` and `exposure` into the `apply_arf` method of the `ARF` class rather than have it in the `apply_response` method in `XSpectrum`, but then I realized that this doesn't work for our simulations because the exposure of the observation and the ARF are not the same!

**Comment by eblur** [comment]
I can't remember why they are different for the fake data.

But yes, sometimes they are different, for reasons that are incomprehensible, but sometimes come up.

The ISIS (Interactive Spectral Interpretation System) rule is to always use the exposure time contained in the ARF (I suppose because that also holds the fracexpo information).

I think that moving exposure and fracexpo into the apply_arf method is reasonable.

**Comment by dhuppenkothen** [comment]
I think they're different for fake data because the ARF and RMF came from a real observation, but the exposure chosen when making the fake spectra was different from the exposure of the real observation. Hence the difference. 

Well, there are several problems with always using the exposure time in the ARF: 
(1) the spectral normalization during fitting will be wrong if the exposure in the ARF is not the correct one (as is the case for our simulations)
(2) the flux spectra will be off, too, by a factor directly related to the difference in exposure time. 

The latter problem was the one that tripped me up when moving exposure and fracexpo into the `apply_arf` method and then trying to make a flux spectrum.

**Comment by eblur** [comment]
Thanks, yes, that's why it is different for fake data.  I'm working on several things at once, so it's hard to go back and forth.

This comes up a lot, so I think we just have to pick one exposure time to use and then be consistent.  In ISIS, it is known that the exposure time from the ARF is always used. When I know they are different, I change the exposure time on the ARF to the value I want.

We can throw a warning (like I did for the bin units) if the ARF exposure does not match the data exposure.  Do you want me to do that?

I personally would like to stick to the standard I'm used to (using the ARF exposure time)

**Comment by dhuppenkothen** [comment]
ok, let's stick to using the exposure in the ARF. Since I'm moving some those calculations into `apply_arf` anyway, I can add a warning.

**Comment by eblur** [comment]
Okay, I'm going to play with this in a different branch to confirm that it is what I think it is.  My hunch is that this is why fracexpo is messing everything up.

**Comment by eblur** [comment]
Nope!  Fracexpo still gums up the works.  Will let you know next week what I find out from HETG folks.
